metis-dereference/metis-dereference-rest/pom.xml

```diff
@@ -67,7 +63,6 @@
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
       <version>${version.servlet.api}</version>
-      <scope>provided</scope>
```

Comment:
Why can't we assume that this is provided by the application container? This also goes for all cases where you remove this scope.

Reply:
Well the new version is not part of tomcat or cloud foundry build pack.
So I figured why risk not having it and not just always have it available.

Reply:
OK, sounds fair.